go/store/prolly/message/prolly_map.go

```diff
@@ -198,13 +206,12 @@ func estimateProllyMapSize(keys, values [][]byte, subtrees []uint64) (keySz, val
 	}
 
 	// todo(andy): better estimates
-	bufSz += keySz + valSz               // tuples
-	bufSz += refCntSz                    // subtree counts
-	bufSz += len(keys)*2 + len(values)*2 // offsets
```

Comment:
need to put this back to account for `[uint16]` offsets. Underestimating the buffer size slows us down a lot